bot/TODO.md

```
@@ -0,0 +1,51 @@
+# TODO List
+
+## ✅ Completed Tasks
+
+### High Priority
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)